telemetry/specs/INT.mdk

```diff
@@ -1194,9 +1212,11 @@ INT Metadata Header and Metadata Stack, followed by TCP payload:
     0                   1                   2                   3
```

Comment:
The length above in the shim header does not seem correct.

Reply:
Fixed.